build/azure-pipelines/common/listNodeModules.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+if (process.argv.length !== 3) {
+	console.error('Usage: node listNodeModules.js OUTPUT_FILE');
+	process.exit(-1);
+}
+
+const ROOT = path.join(__dirname, '../../../');
+
+function findNodeModulesFiles(location: string, inNodeModules: boolean, result: string[]) {
```

Comment:
Same, should be able to use glob here to make this simpler

Reply:
I just copied what vscode was doing. Should I still look into using glob?

Reply:
Eh, if you were copying what they were doing then this is fine I guess. But something to keep in mind in the future since it's a heck of a lot more readable.